src/web/components/Approvals/ApprovalItem.scss

```diff
@@ -0,0 +1,31 @@
+.approval-item {
+  line-height: 2rem;
+
+  .approval-type-label {
+    background: var(--theme-label-background);
+    border-radius: 3px;
+    padding: 4px 8px;
+  }
+
+  .approval-logo {
+    width: 50px;
+    margin-right: 16px;
+  }
+
+  .approval-types {
+    display: flex;
+    align-items: center;
+    font-size: 0.75rem;
```

Comment:
This was `0.625rem` in ParticipantItem.scss which I thought looked too small here.